netket/graph/graph.py

```diff
@@ -0,0 +1,166 @@
+from abstract_graph import AbstractGraph
+
+import numpy as _np
+import networkx as _nx
+
+
+class NetworkX(AbstractGraph):
+    """ Wrapper for a networkx graph"""
+
+    def __init__(self, graph):
+        """
+        Constructs a netket graph from a networkx graph.
+
+        Args: 
+            graph: A networkx graph (might be a networkx.Graph or a networkx.MultiGraph)
+        Examples:
+            A graph of nodes [0,1,2] with edges [(0,1), (0,2), (1,2)]
+            >>> import netket
+            >>> import networkx
+            >>> nx_g = networkx.Graph([(0,1), (0,2), (1,2)])
+            >>> nk_g = netket.graph.NetworkX(nx_g)
+            >>> print(nk_g.n_nodes)
+            3
+        """
+        assert isinstance(graph, _nx.classes.graph.Graph) or isinstance(
+            graph, _nx.classes.multigraph.MultiGraph
+        )
+
+        if isinstance(graph, _nx.classes.graph.Graph):
+            self.graph = _nx.MultiGraph(graph)
```

Comment:
What if `graph` is a multi-graph? Then `self.graph` is never assigned, isn't it?